src/main/resources/assembler.yaml

```diff
@@ -1,5 +1,5 @@
 server:
```

Comment:
These look like the citus ports. I'm cool with this if we're having everyone use citus by default, otherwise just a check that these weren't checked in accidentally

Reply:
Ugh, yes. good catch.